src/components/App/App.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import { Route, Routes } from 'react-router-dom';
 import Home from '../Home/Home';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fab } from '@fortawesome/free-brands-svg-icons';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+import { far } from '@fortawesome/free-regular-svg-icons';
 
-const App = () => (
-  <div className="container">
-    <Routes>
-      <Route exact path="/" element={<Home />} />
-    </Routes>
-  </div>
-);
+function App() {
+  library.add(fab, fas, far);
```

Comment:
I think this can be hoisted outside of the function?  I don' think it matters though since app shouldn't re-render